.github/workflows/CI.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Install dependencies
```

Comment:
Here, I am just introducing NetCDF4 as a dependency for the pip-tests. This is an optional dependency to support writing to a kind of trajectory file.